src/tools/Policy/AzureStack.Policy.psm1

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# See LICENSE.txt in the project root for license information.
+
+#requires -Modules Az.Accounts
+
+<#
```

Comment:
Every cmdlet needs to have example and parameter description so that we can generate the docs with PlatyPs. e.g.
https://github.com/Azure/azurestack-powershell/blob/master/src/Azs.Deployment.Admin/Module/Azs.Deployment.Admin.psm1#L320